spec/index.bs

```diff
@@ -2907,9 +2933,7 @@ dictionary GPUImageBitmapCopyView {
 
         Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUBuffer}} |source|, {{GPUSize64}} |sourceOffset|, {{GPUBuffer}} |destination|, {{GPUSize64}} |destinationOffset|, {{GPUSize64}} |size|, the following validation rules apply:
 
-          - |encoder| must be a [=valid=] {{GPUCommandEncoder}}.
-          - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must not be called when a {{GPURenderPassEncoder}} is active on |encoder|.
-          - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must not be called when a {{GPUComputePassEncoder}} is active on |encoder|.
+          - {{GPUCommandEncoder/[[state]]}} of |encoder| must be [=encoder state/open=].
```

Comment:
nit: `|encoder|.{{GPUCommandEncoder/[[state]]}}`, and the same below.

Reply:
Right! Done.